amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -1557,7 +1522,6 @@ enum ClientType {
   TokenClient
   TokenLockingClient
   VotingReputationClient
-  WhitelistClient
```

Comment:
Hey @iamsamgibbs I have just spoken to Arren so he wants is so that new Contributors who were added via the Manage Verified members flow shouldn't be able to view the Colony right away. They should still have to go through the invite process.

I have amended the block ingestor to reflect this. Can you please re-test this when you have the chance? Thank you! :)

Reply:
@rumzledz WhitelistClient refers to one of the extensions (smart contract) that we do not use anymore, but it might be a good idea to keep it in for legacy purposes.